molecule/azkabanwebsolo/molecule.yml

```diff
@@ -0,0 +1,48 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+  enabled: False
+platforms:
+  - name: azkabanexecutor.vm
```

Comment:
As `.vm` suffix is no longer needed, I would remove it

Reply:
Done